### PR TITLE
SUR-323 - Accessibility fixes for Force UI components.

### PR DESCRIPTION
**Pull request by jaieds**
### Description

- Resolved Accessibility issues.
- Resolved the issue where the Tooltip and the Dropdown weren’t showing.

### Screenshots

- https://d.pr/i/U66gRf

### Checklist:

-   [x] My code is tested
-   [ ] My code passes the PHPCS tests
-   [ ] I've created the npm build.
-   [ ] My code follows accessibility standards 
-   [x] My code has proper inline documentation 
-   [ ] I've included any necessary tests 
-   [ ] I've included developer documentation 
-   [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by JelenaTakac — APPROVED** [review]
Tested.

File: .storybook/preview.tsx [hunk 1]
```diff
@@ -11,6 +11,16 @@ const preview: Preview = {
...
+		a11y: {
+			config: {
+				rules: [
+					{
+						id: 'color-contrast',
+						enabled: false,
+					},
+				],
+			},
+		},
```
Comment (vrundakansara) [inline .storybook/preview.tsx:23]: Any reason why we have disabled this?

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: .storybook/preview.tsx (hunk 1 above)
Comment (jaieds) [reply .storybook/preview.tsx:23]: @vrundakansara, we replaced some colors with new ones Akbar provided for the Color contrast accessibility issue, but it still was showing the color contrast ratio violation.

Akbar suggested ignoring the warning and then we disabled the rule.

File: src/components/breadcrumb/breadcrumb.tsx [hunk 2]
```diff
@@ -111,9 +111,9 @@ export const BreadcrumbSeparator = ( { type }: BreadcrumbSeparatorProps ) => {
 	};
 
 	return (
-		<span className="flex items-center text-text-tertiary mx-2">
+		<li className="flex items-center text-text-tertiary mx-2">
```
[the remark this answers is not in the document]
Comment (jaieds) [reply src/components/breadcrumb/breadcrumb.tsx]: Done

**Comment by jaieds** [comment]
@vrundakansara, Resolved templates audibility issues.

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**